Automattic/wp-calypso
# Signup: Remove unused FSE Beta assignment code

#### Changes proposed in this Pull Request

Now that all new sites created have FSE features turned on, we no longer need any code related to beta assignment by user id. This change removes that code (that was added in https://github.com/Automattic/wp-calypso/pull/60330)

It appears that the user id sign up dependency introduced in https://github.com/Automattic/wp-calypso/pull/60330 may cause issues with mobile signup flows. This removal should resolve that problem (see p1644013502153649-slack-CFFF01Q4V)

#### Testing instructions

* There shouldn't be any change noticed in this PR. Make sure the new user signup and `site-setup` flow works as expected.
